TwilioVerifySDK/TwilioSecurity/Sources/Storage/SecureStorage.swift

```diff
@@ -21,37 +21,44 @@ import Foundation
 
 ///:nodoc:
 public protocol SecureStorageProvider {
-  func save(_ data: Data, withKey key: String) throws
+  func save(_ data: Data, withKey key: String, withServiceName service: String?) throws
   func get(_ key: String) throws -> Data
   func removeValue(for key: String) throws
-  func getAll() throws -> [Data]
-  func clear() throws
+  func getAll(withServiceName service: String?) throws -> [Data]
+  func clear(withServiceName service: String?) throws
 }
 
 ///:nodoc:
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/9cbd540c6135af3d1641ad17aef5ccc5e04d146c/TwilioVerifySDK/TwilioSecurity/Sources/Storage/SecureStorage.swift#L31"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>SecureStorage.swift:31</code></td>
    </tr>
  </tbody>
</table>